Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent {
+    label 'huge && bionic && x86_64 && !xenial && !trusty'
+  }
+
```

Comment:
might want to use https://github.com/jenkinsci/pipeline-github-plugin for its pipeline trigger. see https://github.com/jenkinsci/pipeline-github-plugin#declarative-pipeline

Reply:
It'd be really nice if we didn't have to restrict `make check` to Ubuntu.  Is it still a requirement?

Reply:
not on master at least. we've being running the perf test in RHEL8/CentOS8 for couple months.

at least it does not hurt to enable it and give it a spin, IMHO.

Reply:
```suggestion
  triggers {
    issueCommentTrigger('jenkins test make check')
  }
```

Reply:
@tchaikov that requires [github-pipeline-plugin](https://plugins.jenkins.io/pipeline-github/)

Reply:
@epuertat just intalled.

Reply:
Thank you @tchaikov ! 

Could you please check if the current API Token includes the [permissions as described here](https://github.com/ceph/ceph/pull/37425#issuecomment-716209585).